pkg/cmd/init/options.go

```diff
@@ -17,6 +17,10 @@ type Options struct {
 	useBootstrapToken bool
 	//if true the hub will be reinstalled
 	force bool
+	//Installing release version of OCM
+	version string
```

Comment:
would call it tag @qiujian16 What do you think

Reply:
renamed to tag

Reply:
I would call it version since the tag could be different for images.